src/snowflake/snowpark/mock/mock_plan.py

```diff
@@ -1,388 +0,0 @@
-#
```

Comment:
Moved all code to `mock_select_statement` to resolve the circular dependency that `MockExecutionPlan` references classes defined in `mock_select_statement` and `mock_select_statement` relies on executing `MockExecutionPlan` to retrieve attributes.